bookmarks.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Quiz</title>
+  </head>
+  <body>
+    <h1>Quiz App</h1>
+    <main>
+      <section class="index-card">
+        <h2>Question</h2>
+        <p class="bookmark">
```

Comment:
ich frag mich, ob du das img in ein p Tag packen musst. vielleicht hast du dir was voll schlaues dabei gedacht und ich sehe es nur gerade nicht ;) aber prinzipiell kannst du auch dem img direkt die Klasse geben und dein p Tag weglassen, vermute ich.